HW1.py

```diff
@@ -0,0 +1,104 @@
+wrong_alphabet = 'BbDdEeFfHhIiJjKkLlMmNnOoPpQqRrSsVvWwYyZz'
```

Comment:
Гораздо практичнее задать "правильный" алфавит, так как просто невозможно предусмотреть все возможные неправильные символы. Например: 
```suggestion
true_alphabet = set("ATGCatgcUu")
```